docs/intro/tutorial.rst

```diff
@@ -379,78 +391,67 @@ Let's add this code to our spider::
+<topics-feed-exports>`, with the following command::
 
-    import scrapy
-    from tutorial.items import QuoteItem
+    scrapy crawl quotes -o items.json
 
+That will generate an ``items.json`` file containing all scraped items,
+serialized in `JSON`_.
 
-    class QuotesSpider(scrapy.Spider):
-        name = "quotes"
-        start_urls = [
-            'http://quotes.toscrape.com/page/1/',
-            'http://quotes.toscrape.com/page/2/',
-        ]
+You could've also used other formats, like `JSON Lines`_::
 
-        def parse(self, response):
-            for quote in response.xpath('//div[@class="quote"]'):
-                item = QuoteItem()
-                item['text'] = quote.xpath('span[@class="text"]/text()').extract_first()
-                item['author'] = quote.xpath('span/small/text()').extract_first()
-                yield item
+    scrapy crawl quotes -o items.jl
 
+The `JSON Lines`_ format is useful because it's stream-like, you can easily
+append new records to it. As each record is a separate line, you can also
+process big files without having to fit everything in memory, there are tools
+like `JQ`_ to help doing that at the command-line.
 
-Now crawling quotes.toscrape.com yields ``QuoteItem`` objects::
+In small projects (like the one in this tutorial), that should be enough.
+However, if you want to perform more complex things with the scraped items, you
+can write an :ref:`Item Pipeline <topics-item-pipeline>`. As with Items, a
+placeholder file for Item Pipelines has been set up for you when the project is
+created, in ``tutorial/pipelines.py``. Though you don't need to implement any item
+pipelines if you just want to store the scraped items.
 
-    2016-09-02 16:35:20 [scrapy] DEBUG: Scraped from <200 http://quotes.toscrape.com/page/2/>
-    {'author': 'Oscar Wilde',
-     'text': '“We are all in the gutter, but some of us are looking at the stars.”'}
-    2016-09-02 16:35:20 [scrapy] DEBUG: Scraped from <200 http://quotes.toscrape.com/page/2/>
-    {'author': 'Mark Twain',
-     'text': '“The man who does not read has no advantage over the man who cannot read.”'}
+.. _JSON Lines: http://jsonlines.org
+.. _JQ: https://stedolan.github.io/jq
 
 
 Following links
 ===============
 
 Let's say, instead of just scraping the stuff from the first two pages
-from quotes.toscrape.com, you want quotes from all the pages in the website.
+from http://quotes.toscrape.com, you want quotes from all the pages in the website.
 
-Now that you know how to extract data from a page, why not extract the
-pagination links in each page, follow them and then extract the data you
-want for all of them?
+Now that you know how to extract data from pages, let's see how to follow links
+from them.
 
-Here is a modification to our spider that does just that::
+Here is a modification of our spider that recursively follows the link to the next
+page, extracting data from it::
```

Comment:
I think here you could have the HTML snippet for pagination elements, otherwise it may confuse readers about how `response.css('li.next a::attr("href")')` was chosen.
Also, this instroduces `::attr(href)`, another handy yet non-standard use of CSS selectors. Worth a note I think.